CI/before_script.msvc.sh

```diff
@@ -421,7 +421,7 @@ if [ -z $SKIP_DOWNLOAD ]; then
 		fi
 
 		download "Qt 5.7.0" \
-			"https://download.qt.io/archive/qt/5.7/5.7.0/qt-opensource-windows-x86-msvc${MSVC_YEAR}${QT_SUFFIX}-5.7.0.exe" \
+			"http://download.qt.io/new_archive/qt/5.7/5.7.0/qt-opensource-windows-x86-msvc${MSVC_YEAR}${QT_SUFFIX}-5.7.0.exe" \
```

Comment:
Why do you use http instead of https?

Reply:
Agreed, if possible, switch back to https please.

Reply:
Tested, works with https.

Reply:
Fixed upstream to do not waste a time for rebasing.